### PR TITLE
Add NeSI platforms to Julia supported HPC systems  

### PR DESCRIPTION
**Pull request by DininduSenanayake**
Add NeSI (New Zealand eScience Infrastructure Mahuika  and Maui platforms to Julia supported HPC systems

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,6 +169,8 @@ installation and/or support for using Julia to its users:
...
 [HPC2N, Umeå U](https://www.hpc2n.umu.se/) | [Kebnekaise](https://www.hpc2n.umu.se/resources/hardware/kebnekaise) | ✅ | ✅ | ? | [Intel Xeon Broadwell + Skylake](https://www.hpc2n.umu.se/resources/hardware/kebnekaise) | [Nvidia Tesla K80](https://www.hpc2n.umu.se/resources/hardware/kebnekaise), [Nvidia Tesla V100](https://www.hpc2n.umu.se/resources/hardware/kebnekaise) | [1](https://www.hpc2n.umu.se/resources/software/julia)
+[NeSI](https://www.nesi.org.nz/) | [Mahuika,Māui](https://www.nesi.org.nz/services/high-performance-computing-and-analytics/platforms) | ✅ | ✅ | ✅ | [Intel Xeon Broadwell + Cascaselake + AMD EPYC Milan](https://www.nesi.org.nz/services/high-performance-computing-and-analytics/platforms) | [Nvidia Tesla P100, A100](https://support.nesi.org.nz/hc/en-gb/articles/360001471955) | [1](https://support.nesi.org.nz/hc/en-gb/articles/360001175895-Julia)
```
Comment (sloede) [inline README.md:172]:
```suggestion
[NeSI](https://www.nesi.org.nz/) | [Mahuika, Māui](https://www.nesi.org.nz/services/high-performance-computing-and-analytics/platforms) | ✅ | ✅ | ✅ | [Intel Xeon Broadwell/Cascade Lake + AMD EPYC Milan](https://www.nesi.org.nz/services/high-performance-computing-and-analytics/platforms) | [Nvidia Tesla P100, A100](https://support.nesi.org.nz/hc/en-gb/articles/360001471955) | [1](https://support.nesi.org.nz/hc/en-gb/articles/360001175895-Julia)
```
Fix minor spelling error + whitespace

**Review by sloede — COMMENTED** [review]
Thanks a lot for contributing another Julia HPC system on another continent, @DininduSenanayake! I have made a minor edit suggestion, but other than that it looks ready to be merged!

File: README.md (hunk 1 above)
Comment (DininduSenanayake) [reply README.md:172]: Ooops..Thanks for spotting and fixing the typo @sloede  :slightly_smiling_face:

Comment (sloede) [reply README.md:172]: If you want, feel free to commit the suggestion yourself. Otherwise I can also do it for you and merge the PR afterwards.

Comment (sloede) [reply README.md:172]: And no worries, I am glad you contributed!

**Review by sloede — APPROVED** [review]
LGTM, thanks!

**Pull request merged**